src/App.tsx

```diff
@@ -459,14 +476,50 @@ export class App extends React.PureComponent<unknown, AppState> {
 
 		if (newYearTrackedStats.carbonSavings >= 0.5) {
 			this.setPage(Pages.winScreen);
-		} else if (newYearTrackedStats.year === 10) {
+		} else if (newYearTrackedStats.year === this.state.gameSettings.totalIterations + 1) {
 			this.setPage(Pages.loseScreen);
 		} else {
 			this.setPage(Pages.selectScope);
 		}
 
 	}
 
+	handleGameSettingsOnProceed(totalYearIterations: number){
+		let budget = 0;
+		let naturalGas = 0;
+		let electricity = 0;
```

Comment:
Starting budgets and values are the kind of thing I would store in the game settings/defaults